src/pageServices/contentScripts/highlight.js

```diff
@@ -299,12 +307,27 @@ export const highlightOnPage = () => {
     if (!event.clientX && !event.clientY) return;
   };
 
+  const addToPOListener = () => {
```

Comment:
please, rename this function according to principle "what it's triggered by" instead of "what we expect to do"